### PR TITLE
Provide Jenkinsfile for Jenkins build and test

### PR DESCRIPTION
**Pull request by jasonkatonica**
A new `Jenkinsfile` has been provided to serve as a template for
users who would like to build and test the `OpenJCEPlus` project using
Jenkins.

Signed-off-by: Jason Katonica <katonica@us.ibm.com>
Signed-off-by: Kostas Tsiounis <kostas.tsiounis@ibm.com>
Co-authored-by: Jason Katonica <katonica@us.ibm.com>
Co-authored-by: Kostas Tsiounis <kostas.tsiounis@ibm.com>

### CODE REVIEW

**Review by KostasTsiounis — APPROVED** [review]
LGTM

File: Jenkinsfile [hunk 1]
```diff
@@ -522,8 +522,8 @@ pipeline {
             Build for aarch64_mac platform')
         booleanParam(name: 'x86_64_mac', defaultValue: true, description: '\
             Build for x86-64_mac platform')
-        booleanParam(name: 'aarch64_linux', defaultValue: true, description: '\
-            Build for aarch64_linux platform')
+        booleanParam(name: 'AARCH64_LINUX', defaultValue: true, description: '\
```
Comment (KostasTsiounis) [inline Jenkinsfile:525]: Any particular reason why this is in capital letters?

**Review by JinhangZhang — APPROVED** [review]
LGTM

**Comment by jasonkatonica** [comment]
I have found that when this is build with a new branch the default values were not set correctly. I made updates via https://github.com/IBM/OpenJCEPlus/pull/332/commits/5cf4b2238e1aaee290060846bcea2bb8cbb0f7ed to fix this issue.

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,687 @@
...
+            
+        }
+
+        def java_home = "export JAVA_HOME=$WORKSPACE/java/jdk;"
+        def gskit_home = "export GSKIT_HOME=$WORKSPACE/openjceplus/OCK/jgsk_sdk;"
+        def environment = "export PATH=$WORKSPACE/apache-maven-3.9.6/bin:\$PATH;"
+        def ock_path = "$WORKSPACE/openjceplus/OCK/"
+        if (software == "windows") {
+            def workspace_unix_style = "$WORKSPACE".replace("\\", "/").replace("C:", "/cygdrive/c")
+            environment = """export PATH=${workspace_unix_style}/apache-maven-3.9.6/bin:/cygdrive/c/Program\\ Files\\ \\(x86\\)/Windows\\ Kits/10/bin/10.0.19041.0/x64/:/cygdrive/c/Program\\ Files/Microsoft\\ Visual\\ Studio/2022/Professional/VC/Tools/MSVC/14.31.31103/bin/Hostx64/x64/:\$PATH;\\
```
Comment (taoliult) [inline Jenkinsfile:308]: For this version of VC tools (e.g., 14.31.31103), will the environment always have the same version across all the Windows build platforms?

Comment (jasonkatonica) [reply Jenkinsfile:308]: Hello for now yes all the systems have VC tools in this location. Over time we may have the change this if we decide to use another version. There are no plans at this time to change this.

Comment (taoliult) [reply Jenkinsfile:308]: Ok, got it.

**Review by taoliult — APPROVED** [review]
LGTM

File: Jenkinsfile (hunk 1 above)
Comment (jasonkatonica) [reply Jenkinsfile]: Sorry this was a mistake and was reverted.

**Pull request merged**